instrumented/integration/src/androidTest/kotlin/com/datadog/android/sdk/integration/sessionreplay/ConsentGrantedSrTest.kt

```diff
@@ -25,6 +25,8 @@ internal class ConsentGrantedSrTest : SrSnapshotTest() {
 
     @Test
     fun verifySessionFirstSnapshot() {
+        // we need this to avoid the Bitrise flakiness
+        rule.activity.clickMeButton.performClick()
```

Comment:
well for some reason on Bitrise it doesn't trigger the `onDraw` method when the `Activity` is resumed. Probably because this is happening too fast or for some other reason that I don't really understand. Idea is that I could not reproduce this locally at all so it might be related with the state of that `Emulator` on Bitrise.